### PR TITLE
Hotels: Replace deprecated price types with money type

### LINKED ISSUE #1513: Create field for maxSelectableRooms in graphql
Then use field in `RoomRow.js`

### PR DESCRIPTION
**Pull request by tbergq**
closes #1513

### CODE REVIEW

File: app/hotels/src/map/allHotels/MapView.js [hunk 1]
```diff
@@ -184,7 +184,7 @@ export class Map extends React.Component<Props, State> {
 
   renderHotelMarker = (hotel: Object, index: number) => {
     const { selectedIndex } = this.props;
-    const price = hotel?.price;
+    const price = hotel?.money;
```
Comment (RobinCsl) [inline app/hotels/src/map/allHotels/MapView.js]: It's weird to read `hotel.money`... Could you maybe rename the field in the query instead?

Comment (tbergq) [reply app/hotels/src/map/allHotels/MapView.js]: Yeah, good idea, I see that this is weird! I will do it on monday

Comment (tbergq) [reply app/hotels/src/map/allHotels/MapView.js]: @RobinCsl 🚀

**Comment by tbergq** [comment]
@RobinCsl 🚀

**Pull request merged**